src/estimation.jl

```diff
@@ -12,26 +12,57 @@ export esprit, jacobsen, quinn
 ESPRIT [^Roy1986] algorithm for frequency estimation.
 Estimation of Signal Parameters via Rotational Invariance Techniques
 
-Given length N signal "x" that is the sum of p sinusoids of unknown frequencies,
-estimate and return an array of the p frequencies.
+Given length N signal `x` that is the sum of `p` complex exponentials (cisoids) of unknown frequencies,
```

Comment:
Does ESPRIT make any assumptions about the phase of the cisoids? If so, it may be worth it to mention them here.

Reply:
Yes, good point. How about
```suggestion
Given a signal `x` of length N that is the sum of `p` complex exponentials (cisoids) with unknown constant amplitudes and phases,
```

Reply:
Sounds good!